nbs/api/14_quarto.ipynb

```diff
@@ -50,7 +50,8 @@
     "from fastcore.shutil import rmtree,move,copytree\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.                if not len(val): contents.append({'section': name})
</pre><p>This line will never be called I think. Empty directory will not be included in the <code>glob</code> and is ignored.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/1307/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
Good point, thanks.